admin/static/css/banners.css

```diff
@@ -0,0 +1,5 @@
+.banner-background-color {
+    margin-top: 10px;
+    padding-left: 5px;
+    padding-right: 5px;
+}
```

Comment:
So when I was trying this out, I grabbed a random SVG and popped it in there and it was waaaaaay too big - perhaps adding a `max-height` style property to the image component would help prevent people from making silly mistakes like I did?

<img width="1163" alt="screen shot 2018-01-23 at 12 19 55 pm" src="https://user-images.githubusercontent.com/801594/35290552-9b9f63c6-0038-11e8-8f7a-b983b7836124.png">

Reply:
That's a good point. I think I'm going to add validation on the uploader with max height/width so that we aren't storing images that are bigger than we need.

Reply:
Ah, but you can still restrict its height using CSS -- I think this is actually pretty important just to prevent any accidental things -- something like setting the `max-height` property in `banners.css` would do the trick

<img width="1250" alt="screen shot 2018-01-29 at 12 13 38 pm" src="https://user-images.githubusercontent.com/801594/35524090-73abfe3c-04ee-11e8-9626-d97241d94345.png">